# Message header
Header header

int8[]    type
float64[] distance
float64[] direction
float64[] risk_value
float64[] speed
float64[] orientation
float64[] size_x
float64[] size_y